Docs.md

```diff
@@ -226,7 +226,7 @@ You should use this class for all of your Thorn tests except those which test th
 
 ### Creating Agents
 
-An Agent corresponds directly to a user that exists in the SugarCRM instance you are testing. That user *must* exist in the database before you attempt to create a user agent for it. You can use the [Thorn Fixtures API](#thorn::fixtures) to create any needed users.
+An Agent corresponds directly to a user that exists in the SugarCRM instance you are testing. All users except for the admin user specified by the `THORN_ADMIN_USERNAME` environment variable *must* be created by `Fixtures.create` before you attempt to create a user agent for it. Please refer to [Thorn Fixtures API](#thorn::fixtures) to create the needed users.
```

Comment:
@rhoggSugarcrm As discussed offline, the username that is automatically registered by Thorn is not necessarily the OOTB admin, it's the one that's passed in.

Looking at Thorn in isolation, (i.e. outside of the gulp task, which lives in Mango, not in Thorn), the only access that Thorn has to the admin username and password is the environment.

It might be worth bringing up the `THORN_ADMIN_USERNAME` and `THORN_ADMIN_PASSWORD` unless we plan on supplying the gulpfile for our Thorn consumers, especially since we've now opened Thorn up.